equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/repository/TaskRepository.java

```diff
@@ -1,12 +1,11 @@
 package uk.gov.justice.laa.crime.equinity.historicaldata.repository;
 
-import java.util.List;
-
 import org.springframework.data.jpa.repository.JpaRepository;
 
-import uk.gov.justice.laa.crime.equinity.historicaldata.model.Tasks;
+import uk.gov.justice.laa.crime.equinity.historicaldata.model.Task;
+
 
-public interface TasksRepository extends JpaRepository<Tasks, Long> {
+public interface TaskRepository extends JpaRepository<Task, Long> {
   //  List<Tasks> findByID(Long ID);
```

Comment:
Pls remove commented code

Reply:
I will removed on the next PR with the new changes